integrations/axum/src/extract.rs

```diff
@@ -97,17 +91,14 @@ impl<R> GraphQLBatchRequest<R> {
 }
 
 #[async_trait::async_trait]
-impl<S, B, R> FromRequest<S, B> for GraphQLBatchRequest<R>
+impl<S, R> FromRequest<S> for GraphQLBatchRequest<R>
 where
-    B: HttpBody + Send + Sync + 'static,
-    B::Data: Into<Bytes>,
-    B::Error: Into<BoxError>,
     S: Send + Sync,
     R: IntoResponse + From<ParseRequestError>,
 {
     type Rejection = R;
 
-    async fn from_request(req: Request<B>, state: &S) -> Result<Self, Self::Rejection> {
+    async fn from_request(req: Request, _state: &S) -> Result<Self, Self::Rejection> {
```

Comment:
Not caught by the existing tests, but it fails for me:

```
102 |         if let (&Method::GET, uri) = (req.method(), req.uri()) {
    |                                       --- has type `&axum::http::Request<Body>` which is not `Send`
...
128 |                 .await?,
    |                  ^^^^^ await occurs here, with `req` maybe used later
...
132 |     }
    |     - `req` is later dropped here
help: consider moving this into a `let` binding to create a shorter lived borrow
```

A solution is to check the method without binding:

```rust
if req.method() == Method::GET {
    let uri = req.uri();
    …
```